test/test_helper.rb

```diff
@@ -1,3 +1,13 @@
+# Coverage - Keep these two lines at the top of this file
+require 'simplecov'
+require 'coveralls'
+SimpleCov.formatters = [SimpleCov::Formatter::HTMLFormatter, Coveralls::SimpleCov::Formatter]
+SimpleCov.start do
+  minimum_coverage 52
+  refuse_coverage_drop
+  track_files "{lib,test}/**/*.rb"
```

Comment:
Are you sure? I am thinking this causes simplecov to check that we have tests for our tests.